tests/unit/net_timeout/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
Could you move the net_timeout test under `tests/unit/net/` directory, there is already another network related test there.

Reply:
? There is no `tests/unit/net` parent directory.  (Note this is a unit test, not a platform test).

Reply:
I have no objection to making it a subdirectory, though, if there will be additional unit tests.  (There probably should....)

Reply:
Hmm, indeed. I had some old directory in my tree and did not look what was in it, nevermind then.